tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -573,6 +575,11 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
     );
 
     this.isPinned$ = this.store.select(getCardPinnedState, this.cardId);
+
+    this.rangeEnabled$ = this.store.select(
```

Comment:
This is good to consider. It does seem cleaner to pull this from the state inside the reducer. However, the way I built it allows for more reuse(One day we may want to open to a certain tab without coming from a card and this action could be used there too). Since I am a bit up in the air about it I am going to just leave it as is.